src/plugins/node/prof/compat-inl.h

```diff
@@ -0,0 +1,381 @@
+/*******************************************************************************
+ * Copyright 2014, StrongLoop Inc.
```

Comment:
@rmg can/should we change this? It's IBM code, we own it, we can put any header we want, can't we replace the legacy StrongLoop header with the standard appmetrics one (https://github.com/RuntimeTools/appmetrics/blob/master/src/appmetrics.cpp#L1-L15)?